src/components/Create/PayModsForm.tsx

```diff
@@ -51,7 +51,6 @@ export default function PayModsForm({
 
       <Form layout="vertical">
         <FormItems.ProjectPayoutMods
-          name="mods"
```

Comment:
This doesn't need a name. Removing it fixes a console error:

```
Warning: [antd: Form.Item] `children` is array of render props cannot have `name`. 
```

<img width="658" alt="Screen Shot 2021-11-29 at 2 40 07 pm" src="https://user-images.githubusercontent.com/94939382/143812273-e3dcf10f-1b85-4be1-bce3-bc6d25eea1eb.png">